lib/Minz/ExtensionManager.php

```diff
@@ -26,6 +26,10 @@ final class Minz_ExtensionManager {
 			'list' => array(),
 			'signature' => 'OneToOne',
 		),
+		'entry_auto_read' => array(	// function(FreshRSS_Entry $entry, string $why) -> none
+			'list' => array(),
+			'signature' => 'PassArguments',
+		),
```

Comment:
use [] for arrays ;)

Reply:
Another time, not to mess the rest of the PR here

Reply:
https://github.com/FreshRSS/FreshRSS/pull/5504